filesystem/licenses/BSD

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2009 The Go Authors. All rights reserved.
```

Comment:
This is a specific instance of the BSD license (it has been specialised with "The Go Authors" and "Google").  Might need to include the specific instance for each package that we use.